app/test-utils.ts

```diff
@@ -32,7 +32,8 @@ export const provideRunId = <T>(
 export const printErrors = <T>(testFn: () => Promise<T>, runId?: string): (() => Promise<T>) => {
   return async () => {
     try {
-      return await testFn();
+      const a = await testFn();
```

Comment:
why like this?

Reply:
The only way to currently debug stuff if something went wrong is to lace the codebase with `console.logs` everywhere